lib/daru/io/io.rb

```diff
@@ -221,6 +227,25 @@ def from_csv_hash(path, opts)
 
         headers.each_with_index.map { |h, i| [h, csv_as_arrays[i]] }.to_h
       end
+
+      def from_html_get_list(page)
+        page.search('table').map { |table|
+          index = table.search('th').map { |header| header.text.strip }
+          data = table.search('tr').map { |row| row.search('td').map { |val| val.text.strip } }.keep_if { |a| a!=[] }
+          {'index' => index, 'data' => data}
+        }.keep_if { |a| a['index'] != [] }
+      end
+
+      def from_html_get_df_list(list)
+        list.map { |ele|
+          begin
+            indexes, data = ele['index'], ele['data'].transpose
+            hash = Hash[indexes.map { |index| [index,data[indexes.index(index)]] }]
+            Daru::DataFrame.new(hash)
```

Comment:
I'll have a look at the other constructing methods, and change this accordingly.